.github/workflows/cpp-packaging.yml

```diff
@@ -154,14 +159,14 @@ jobs:
         with:
           repository: nico/demumble
           path: demumble-src
-          ref: v${{ env.demumbleVer }}
+          ref: ${{ env.demumbleVer }}
 
       - name: build demumble
         run: |
           cd demumble-src
           cmake .
           cmake --build .
-          python demumble_test.py
+          python3 demumble_test.py
```

Comment:
Is there a reason this needs to be `python3` here? I thought we were going with `python` from now on?